flutter/third_party/accessibility/ax/ax_event_generator.h

```diff
@@ -125,9 +125,14 @@ class AX_EXPORT AXEventGenerator : public AXTreeObserver {
     const EventParams& event_params;
   };
 
-  class AX_EXPORT Iterator
-      : public std::iterator<std::input_iterator_tag, TargetedEvent> {
+  class AX_EXPORT Iterator {
```

Comment:
Could you please explain a bit more? I can't find the rationale for this change.

Reply:
`std::iterator` is deprecated and generates a deprecation warning (which is treated as an error). I referred to the current [Chromium source code](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/ui/accessibility/ax_event_generator.h) (specifically [this commit](https://chromium.googlesource.com/chromium/src/+/c40a8c4a15034b783944bed298cc1151c4e822d5)) to minimize the use of `#pragma`.

c.f. I couldn't find a replacement for `std::codecvt_utf8` in `Utf8ToUtf32CodePoint`.

Reply:
flutter upstream still use [std::iterator](https://github.com/flutter/engine/blob/52d05954815da75f492c94c70607dc4a8d6cd7a8/third_party/accessibility/ax/ax_event_generator.h#L129), are these differences okay?

Reply:
> c.f. I couldn't find a replacement for std::codecvt_utf8 in Utf8ToUtf32CodePoint.

I think we need to write own one or it just be maintained until the standard provides an appropriate alternative.

Reply:
Should be no problem. They will eventually update their code too.